javascript/javascript1/week3/notes.js

```diff
@@ -0,0 +1,58 @@
+var notes = [];
+
+function saveNote(content, id){
+    if(typeof(id) === "number" && typeof(content) === "string" && content !=""){
+        notes.push({content, id});
+    } else{
+        return "Incorrect note";
+    }
+}
+
+function getNote(id){
+    if(typeof(id) === "number"){
+        for (let i = 0; i < notes.length; i++) {
+            if(notes[i].id === id){
+                return notes[i];
+            }
+            return "Error! Incorrect number."
+        }
+    } else{
+        return "Error!"
+    }
+}
+
+function logOutNotesFormatted() {
+    for (let i = 0; i < notes.length; i++) {
+        console.log(`The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}`);
+    }
+}
+
+
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+const firstNote = getNote(1);
```

Comment:
If I try to `getNote(2)`, it doesn't work! 🐛 Can you see what goes wrong and fix it? 🔍